lib/src/utils/models/file_info.dart

```diff
@@ -0,0 +1,18 @@
+
+import 'package:equatable/equatable.dart';
+
+class FileInfo with EquatableMixin {
+  final String fileName;
+  final String filePath;
+  final int fileSize;
+  final Stream<List<int>>? readStream;
```

Comment:
why we need this?

Reply:
readStream is needed because for convenience. for example, FilePicker, don't have to write to file, but can return as Stream<List<int>> immediately.

Reply:
> FilePicker, don't have to write to file, but can return as Stream<List> immediately.

please dont forget to close the stream in that case

Reply:
no. I can't close it, FilePicker will handle it in that case.